test/mocks/components/metrics.ts

```diff
@@ -0,0 +1,11 @@
+import { IMetricsComponent } from '@well-known-components/interfaces'
+
+export const mockMetrics: jest.Mocked<IMetricsComponent<string>> = {
```

Comment:
Just a heads up about the existence of `createTestMetricsComponent` (_from WKC/metrics package_) maybe is already enough for these suites :)